packages/web/src/scss/components/Header/README.md

```diff
@@ -193,57 +202,154 @@ Use `Header__text` class if you need to display text instead of link:
 </ul>
 ```
 
-### Dialog
+## Header Dialog
 
-Dialog is an optional enhancement of the responsive actions pane. It looks and
-behaves the same on small screens, but it keeps the off-canvas appearance also
-on big screens (i.e. from the desktop breakpoint up).
+Header Dialog is Spirit's solution for responsive navigation and selected use
+cases such as the user menu. Please note Header Dialog is not intended to be
+used for second-level navigation in general.
 
-Dialog consists of almost exactly the same code as the actions pane. The only
-difference is the class of the wrapping `<nav>`:
+Header Dialog implements the [`<dialog>`][mdn-dialog] element to easily provide
+the best possible user experience and accessibility.
 
 ```html
-<nav id="my-header-desktop-dialog" class="Header__dialog">
-  <div class="Header__content">
-    <!-- Close button with optional primary and secondary action slots -->
+<dialog id="my-header-dialog" class="HeaderDialog" aria-label="Menu">
+  <div class="HeaderDialog__panel">
+    <div class="HeaderDialog__content">
+        <!-- Close button with primary and secondary action slots -->
```

Comment:
```suggestion
      <!-- Close button with primary and secondary action slots -->
```
Indentation fix.

Reply:
It seems correct to me on my machine… 🤷🏻‍♂️ 

<img width="685" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/198995975-e1d1f3bb-6f43-4978-959e-77ad5c178f09.png">